apps/projects/app/components/Content/Issues.js

```diff
@@ -228,10 +228,19 @@ class Issues extends React.PureComponent {
     </StyledIssues>
   )
 
+  getExpLevels = () => {
+    let expLevels = []
+    let a = this.props.bountySettings.expLevels.split('\t')
+    for (let i = 0; i < a.length; i += 2)
```

Comment:
Maybe it is not really clear why the step should be 2 good place to comment what is going on here

Reply:
I'll improve it when expLevels manipulation moves to where it should be.